### ISSUE TITLE
newbie documentation needed!  - How to run the examples.... 

### ISSUE THREAD

**Issue by perrinmeyer** [JuliaPlots/MakieGallery.jl#26]
Now that I have compiled glfw3 from scratch, and also sorta figured out PackageCompiler, so Makie loads in under 10 seconds...

I CANT FIGURE OUT HOW TO GENERATE ANY OF THESE FANTASTIC EXAMPLES....(i'm slightly embarrassed to admit -- but Makie uses some serious Julia-FU, way beyond my current understanding...)  

julia> using MakieGallery 
julia> MakieGallery.eval("snake.jl")
"snake.jl"
NOPE

julia> MakieGallery.eval_example("snake.jl")
NOPE

ERROR: type String has no field toplevel
Stacktrace:
 [1] getproperty(::Any, ::Symbol) at ./sysimg.jl:18
 [2] #print_code#42(::String, ::String, ::String, ::Bool, ::getfield(MakieGallery, Symbol("##45#49")), ::getfield(MakieGallery, Symbol("##46#50

**Comment by asinghvi17** [comment]
I don't think MakieGallery was intended to be used in the REPL :)...you could, of course, poke around in the source code, but I think the best way to replicate examples would be to look at the latest docs.  If you put in `using Makie` before each one, and ignore the 'record' bits at the end of the interactive examples, you should be able to replicate the examples.  The docs are generated from MakieGallery.

**Comment by perrinmeyer** [comment]
thanks, I don't feel quite so stupid now....

**Comment by SimonDanisch** [comment]
It was actually intended to be used in the repl :D Just never documented and polished it (so still no need to feel stupid^^)
You can use it like this:
```julia
julia> using MakieGallery
julia> database = MakieGallery.load_database();
julia> database[1].title # can be used e.g. for filter(x-> x.title == "...", database)
"Tutorial simple scatter"
julia> database[1] # pretty printing :) 
...
 x = rand(10)
 y = rand(10)
 colors = rand(10)
 scene = scatter(x, y, color = colors)
...
julia> using Makie
julia> MakieGallery.eval_example(database[1]) # run it!

**Comment by asinghvi17** [comment]
Fixed by #29, functionality added in #33 as well.
